### PR TITLE
Copy changes and fix for broken tests

### PR DESCRIPTION
**Pull request by TobyRet**

### CODE REVIEW

**Comment by TobyRet** [comment]
[Copy changes](https://trello.com/c/uw2kHzQq/414-copy-changes)

File: app/views/search/search.html.erb [hunk 1]
```diff
@@ -1,15 +1,7 @@
-<% content_for :page_title do %>
-  Results for "<%= @query %>" - Find Data Alpha
-<% end %>
-
```
Comment (maxf) [inline app/views/search/search.html.erb]: Why has the page title disappeared?

Comment (TobyRet) [reply app/views/search/search.html.erb]: @maxf - I believe this is dead code copies over by accident from alpha
It's removal does not have a side effect

Comment (TobyRet) [reply app/views/search/search.html.erb]: <% content_for :page_title do %>
  Data.gov.uk - Find Data
<% end %>

is set in application.html.erb

Comment (TobyRet) [reply app/views/search/search.html.erb]: Ah I see now, my bad. Will change

Comment (maxf) [reply app/views/search/search.html.erb]: That code sets the `<title>` for the search results page (ie, 'search results for ...'. If we don’t include it, it’ll have the default title for all pages, which is an accessibility issue.

**Pull request merged**